client/src/containers/SignIn/utils.js

```diff
@@ -1,17 +1,18 @@
 import callApi from '../../libs/axios';
 
-export const handleSubmit = async (e, user, history) => {
+export const handleSubmit = async (e, user, history, dispatch) => {
   e.preventDefault();
   const { email, password } = user;
   const result = await callApi('post', '/user/login', { email, password });
   if (result.data) {
     const { Data } = result.data;
```

Comment:
Data in capital letters? variable names should be in small camelCase form while Components should be in upperCase Form.

Reply:
changed to `data`